.travis.yml

```diff
@@ -13,9 +13,6 @@ matrix:
     - python: 2.7
       env: TOXENV=py27
 
-    - python: 3.4
```

Comment:
I dislike the fact that this PR introduces the dropping of Python 3.4. I see in other jobs that there's an issue with `pandas` ; so I'll try to find a way to solve this incompatibility more gracefully, and see if this sunsetting of Python 3.4 is necessary.
I'll keep you posted.

Reply:
@brunobord good job!